src/account_manager.rs

```diff
@@ -739,32 +732,28 @@ impl AccountManager {
             self.storage_folder.join(STRONGHOLD_FILENAME)
         };
 
-        if storage_path.exists() {
-            let destination = if let Some(filename) = storage_path.file_name() {
-                let destination = if destination.is_dir() {
-                    destination.join(backup_filename(filename.to_str().unwrap()))
-                } else {
-                    destination
-                };
-                let res = fs::copy(storage_path, &destination);
-
-                let mut stronghold_storage = crate::storage::stronghold::StrongholdStorageAdapter::new(
-                    &self.storage_folder.join(STRONGHOLD_FILENAME),
-                )
-                .unwrap();
-                for account_handle in self.accounts.read().await.values() {
-                    stronghold_storage.remove(&account_handle.read().await.id()).await?;
-                }
-
-                res?;
-                destination
+        let destination = if let Some(filename) = storage_path.file_name() {
+            let destination = if destination.is_dir() {
+                destination.join(backup_filename(filename.to_str().unwrap()))
             } else {
-                return Err(crate::Error::StorageDoesntExist);
+                destination
             };
-            Ok(destination)
+            let res = fs::copy(storage_path, &destination);
+
+            let mut stronghold_storage = crate::storage::stronghold::StrongholdStorageAdapter::new(
+                &self.storage_folder.join(STRONGHOLD_FILENAME),
+            )
+            .unwrap();
```

Comment:
Can't this fail?

Reply:
the stronghold adapter never fails; i'll add a note in there